### PR TITLE
Adds Medical Beamgun to Prolathe

### PR DESCRIPTION
**Pull request by Robertha89**
Even maxing it using [ADMIN\] Maximize Research Levels won't be enough to unlock this baby, but any scientist who knows science can do this.

Also, next to impossible to do without help from other departments, unless RNG favors you.

What this means is, not every round will have one, as you need to max science beyond what it recognizes as max, it also gives science a way to mass produce them if a gamma ert or nukeops is stupid enough to drop it on the floor.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
I'm...extremely skeptical on this.

**Comment by Robertha89** [comment]
I set it up to prevent skepticism, cmo access to open it, and the only way to get biotech 6 is by research ing a nymph. The only thing to really change is the resource costs, and if you get any resources back when researching it. Also, i need to be accessible in game, by normal means, to make Uberchargers work, when someone gets around to doing that.

**Comment by Earthdivine** [comment]
If it gets abused we can always just remove it, anywas.

**Comment by Robertha89** [comment]
Which I hope not, this is honestly one of the more interesting items, it will be super amusing when CMO decided to give everyone in medbay one.

File: code/modules/research/designs/medical_designs.dm [hunk 1]
```diff
@@ -198,6 +198,19 @@
...
 	category = list("Medical")
 
+/datum/design/medibeam
+	name = "Medical Beamgun"
+	desc = "Delivers medical nanites in a focused beam."
+	id = "medi_beam"
+	req_tech = list("combat" = 6, "materials" = 7, "powerstorage" = 8, "programming" = 7, "biotech" = 6)//This is literally the highest you can get it in a normal round
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 5000, MAT_GLASS = 4000, MAT_URANIUM = 8000, MAT_DIAMOND = 2000)
+	reliability = 100
```
Comment (Fox-McCloud) [inline code/modules/research/designs/medical_designs.dm]: no need for this.

File: code/modules/projectiles/medbeam.dm [hunk 2]
```diff
@@ -2,6 +2,7 @@
 	name = "Medical Beamgun"
 	desc = "Delivers medical nanites in a focused beam."
 	icon = 'icons/obj/chronos.dmi'
+	origin_tech = "combat=6;materials=7;powerstorage=8;programming=7;biotech6"//The ultimate within medical design
```
Comment (Fox-McCloud) [inline code/modules/projectiles/medbeam.dm]: Yeah, no.

This allows you to hit materials 8 and programming 8; these should all be 1 tech level lower than they are---there's intentional inconsistencies beteween origin tech and tech required to actually produce the item.

**Comment by Fox-McCloud** [comment]
Here's why I'm skeptical:

- It's free, unlimited, effortless healing that even heals bones.
- Unlike the wormhole projector, where heads of staff will likely be uneasy about handing that out for security purposes...there's not much reason for this to NOT be handed out like literal candy
- this is only available to regular crew on gamma
- It's a very expensive item for nukies---some things just aren't meant to be touched by the crew, generally.

- I'm pretty sure the guy who designed this intended to make an R&D varient of this that had some pretty serious downsides to it.

**Comment by Regen1** [comment]
:-1: 

See my post
http://nanotrasen.se/phpBB3/viewtopic.php?f=12&t=6696

**Comment by Fox-McCloud** [comment]
This didn't pass on TG either---someone is likely to code a backpack variety, with some downsides---that sounds hair bit more reasonable, but overall--not everything that gets added needs to necessarily be crew accessible.

**Comment by TheDZD** [comment]
This is incredibly powerful for how simple it is to build (miners actually do their jobs on a regular basis now, since they get nice toys for doing so). RnD can already replace every single department as-is, and with a chem machine + medigun, they can replace almost the entirety of medbay with absolutely zero effort. The most OP department on the station has enough shiny toys as-is. :-1:

**Pull request closed without merging**